UnitedIncome/components
# [APP-2943] - Fix mcg and expandcollapse animations

### Description
Fixes McG and ExpandCollapse animation quirks. McG should no longer stutter and ExpandCollapse should no longer animate when it's supposed to be open by default.

### Testing Instructions
1. Make sure ExpandCollapse and Cards function and animate like normal
1. Pull down `spending-curve-env` client branch. 
1. Navigate to the RealEstate flow and make your way through the flow, see if it has any issues with the animations especially in the Mortgage Payment Info and Housing Costs cards. Test out the flow in Essentials as well

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-2943)
